lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -437,8 +450,6 @@ def add_network_token_cryptogram_and_eci(post, payment_method, options)
       end
 
       def add_digital_wallet(post, payment_method, options)
-        source = payment_method.respond_to?(:source) ? payment_method.source : options[:wallet_type]
```

Comment:
Removed this since `options[:wallet_type]` is only passed for pan only GooglePay which are CreditCards in AM and should be treated as credit cards and not Network Tokens